scripts/run.py

```diff
@@ -154,6 +154,23 @@ def main(config: ConfigBase):
     # Initialise the model with mup
     named_params = list(model.named_parameters())
     param_names = {name for name, _ in named_params}
+
+    modules_without_init = set()
+    for module_name, module_type in model.named_modules():
+        if isinstance(module_type, torch.nn.LayerNorm) or isinstance(module_type, torch.nn.BatchNorm1d) or isinstance(module_type, torch.nn.BatchNorm2d) or isinstance(module_type, torch.nn.BatchNorm3d):
```

Comment:
Here you can use torch.nn.modules.batchnorm._BatchNorm instead of all the BN subclasses.